src/NuGet.Protocol.Catalog/Models/AlternatePackage.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class AlternatePackage
+    {
+        [JsonProperty("@id")]
+        public string Url { get; set; }
+
+        [JsonProperty("@type")]
+        public string Type { get; set; }
```

Comment:
What Type an AlternatePackage can have?

Reply:
It's one of the strange RDF URLs that is the current URL plus a fragment like `#alternatePackage`. It is unused by the client and is there as a coincidence of JSON-LD.